openedx/core/djangoapps/course_roles/toggles.py

```diff
@@ -0,0 +1,22 @@
+"""
+Toggles for Course Roles app.
+"""
+from edx_toggles.toggles import WaffleFlag
+
+# .. toggle_name: FLAG_USE_PERMISSION_CHECKS
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Enabling the toggle will allow the db checks for a users permissions. These are used alongside current
+#   roles checks. If the flag is not enabled, only the roles checks will be used.
+# .. toggle_use_cases: temporary
+# .. toggle_creation_date: 2023-10-17
+# .. toggle_target_removal_date: 2023-12-01
+# .. toggle_warning:
+USE_PERMISSION_CHECKS_FLAG = WaffleFlag('course_roles.use_permission_checks', module_name=__name__) or False  # lint-amnesty, pylint: disable=toggle-missing-annotation
```

Comment:
Why is this pylint exemption necessary?

Reply:
the exception was necessary because the toggle name declared in the docstring was different from the one passed to the constructor, I will modify it to be consistent